radixdlt/src/main/java/com/radixdlt/consensus/Counters.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.consensus;
+
+import java.util.EnumMap;
+import java.util.Map;
+
+/**
+ * Consensus event counting utility class.
+ * Class is NOT thread-safe so must be run in the same thread or with correct locking.
```

Comment:
It would be really super simple to make this thread-safe by just replacing the EnumMap with a ConcurrentMap of some kind.  I think this is really worth doing here.